curricula/templates/curricula/curriculum.html

```diff
@@ -217,6 +221,33 @@ <h2>{{ topic.name }}</h2>
                 });
                 */
             });
+            $("#clone_this").click(function () {
+                $('#clone_this').html("Cloning...").removeClass('btn-warning btn-success').addClass('btn-primary');
+                var pk = $(this).attr('data-pk');
+                var type = $(this).attr('data-type');
+                var children = $('#clone_children').prop('checked');
+                $('#clone_spinner').addClass('fa fa-cog fa-spin');
+
+                $.ajax({
+                     type: "POST",
+                     url: "/clone/",
+                     data: {pk: pk, type: type, children: children},
+                     timeout: 9999999
```

Comment:
this should probably instead either be left blank for an infinite timeout, or given an actual value

Reply:
Noted, I'll hit that when I factor this all out